config.local.yml

```diff
@@ -68,4 +68,3 @@ rest_runner:
     translation_enabled: True
     translation_model: facebook/seamless-m4t-v2-large # facebook/seamless-m4t-v2-large
     translation_device: cpu # cpu | cuda
-    translation_method: segmented # segmented | full
```

Comment:
Was the idea not to add a default instead of removing the choice? Or am I just recalling it wrong?